pom.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Not sure I understand this one. I triggered a Source->Format in Eclipse just to be sure. However, there does not appear to have been any changes with respect to indentation.